webapp/src/pages/Settings.tsx

```diff
@@ -422,15 +440,17 @@ const Settings: React.FC = () => {
               </FormGroup>
             </FormControl>
             <FormControl>
-              {displayPostprocessorToggleSection(pipelineIndex, pipeline)}
+              {pipeline.postprocessors &&
+                displayPostprocessorToggleSection(pipelineIndex, pipeline)}
```

Comment:
Wouldn't that defy the purpose of the checkbox? When `pipeline.postprocessors` is `null`, we want the checkbox to be unchecked (as it is already), not hidden.
```suggestion
              {displayPostprocessorToggleSection(pipelineIndex, pipeline)}
```

Reply:
Oh right! this is not necessary here. I will revert back to the old change. thanks.